Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/workbench/RetrieveProjectDAO.java

```diff
@@ -86,6 +86,29 @@ public ProjectDTO getProjectDTO(String projectId,String userId) throws QuadrigaS
 	}
 
 
+	/**
+	 * Meghana
+	 */
+	@SuppressWarnings("unchecked")
+	@Override
+	public List<ProjectDTO> getAllProjectsDTOByAccessibility(String accessibility) throws QuadrigaStorageException 
+	{

```

Comment:
please use the code style we use
https://diging.atlassian.net/wiki/pages/viewpage.action?pageId=8781898